diff -r 0c7ee56526ac unsupported/Eigen/src/IterativeSolvers/DGMRES.h
--- a/unsupported/Eigen/src/IterativeSolvers/DGMRES.h	Thu Jun 07 15:56:43 2018 +0200
+++ b/unsupported/Eigen/src/IterativeSolvers/DGMRES.h	Tue Aug 14 11:21:09 2018 +0200
@@ -173,7 +173,7 @@
   /** 
    * Set the restart value (default is 30)  
    */
-  Index set_restart(const Index restart) { m_restart=restart; }
+  void set_restart(const Index restart) { m_restart=restart; }
   
   /** 
    * Set the number of eigenvalues to deflate at each restart 
