From 8bf24e2372b142f1a2189ad951fa396c5a4e0e74 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Sat, 1 Aug 2020 09:25:01 +0200
Subject: [PATCH] Fix build with zxing 1.1.0

BUG: 424853
---
 src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 21c0511..d8eee7c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -90,6 +90,7 @@ set(kitinerary_lib_srcs
     stringutil.cpp
 )
 qt5_add_resources(kitinerary_lib_srcs extractors/extractors.qrc)
+kde_source_files_enable_exceptions(barcodedecoder.cpp qimagepurebinarizer.cpp)
 ecm_qt_declare_logging_category(kitinerary_lib_srcs HEADER logging.h IDENTIFIER KItinerary::Log CATEGORY_NAME org.kde.kitinerary)
 ecm_qt_declare_logging_category(kitinerary_lib_srcs HEADER compare-logging.h IDENTIFIER KItinerary::CompareLog CATEGORY_NAME org.kde.kitinerary.comparator)
 ecm_qt_declare_logging_category(kitinerary_lib_srcs HEADER validator-logging.h IDENTIFIER KItinerary::ValidatorLog CATEGORY_NAME org.kde.kitinerary.extractorValidator)
-- 
GitLab

