--- kitinerary-18.08.3/src/pdfdocument.cpp.orig	2018-12-07 21:43:58.668133222 +0100
+++ kitinerary-18.08.3/src/pdfdocument.cpp	2018-12-07 21:44:27.066135935 +0100
@@ -306,7 +306,7 @@
     const auto pageRect = d->m_doc->m_popplerDoc->getPage(d->m_pageNum + 1)->getCropBox();
     std::unique_ptr<GooString> s(device->getText(ratio(pageRect->x1, pageRect->x2, left), ratio(pageRect->y1, pageRect->y2, top),
                                                  ratio(pageRect->x1, pageRect->x2, right), ratio(pageRect->y1, pageRect->y2, bottom)));
-    return QString::fromUtf8(s->getCString());
+    return QString::fromUtf8(s->c_str());
 #else
     Q_UNUSED(left);
     Q_UNUSED(top);
@@ -440,7 +440,7 @@
         PdfPage page;
         page.d->m_pageNum = i;
         page.d->m_doc = doc->d.get();
-        page.d->m_text = QString::fromUtf8(s->getCString());
+        page.d->m_text = QString::fromUtf8(s->c_str());
         page.d->m_images = std::move(device->m_images);
         for (auto it = page.d->m_images.begin(); it != page.d->m_images.end(); ++it) {
             (*it).d->m_page = page.d.data();
