--- kdevplatform-5.0.3/util/kdevplatform_shell_environment.sh.orig	2017-02-16 23:27:05.064582277 +0100
+++ kdevplatform-5.0.3/util/kdevplatform_shell_environment.sh	2017-02-16 23:27:20.618581720 +0100
@@ -56,7 +56,7 @@
 }
 
 # Check if all required tools are there (on the host machine)
-checkToolsInPath sed qdbus ls cut dirname mktemp basename readlink hostname
+checkToolsInPath sed qdbus-qt5 ls cut dirname mktemp basename readlink hostname
 
 if ! [ "$KDEV_SSH_FORWARD_CHAIN" ]; then
     # Check for additional utilities that are required on the client machine
@@ -65,11 +65,11 @@
 
 # Queries the session name from the running application instance
 function getSessionName {
-    echo "$(qdbus $KDEV_DBUS_ID /org/kdevelop/SessionController org.kdevelop.SessionController.sessionName)"
+    echo "$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/SessionController org.kdevelop.SessionController.sessionName)"
 }
 
 function getSessionDir {
-    echo "$(qdbus $KDEV_DBUS_ID /org/kdevelop/SessionController org.kdevelop.SessionController.sessionDir)"
+    echo "$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/SessionController org.kdevelop.SessionController.sessionDir)"
 }
 
 function getCurrentShellEnvPath {
@@ -256,7 +256,7 @@
 
 # Opens a document in internally in the application
 function openDocument {
-    RESULT=$(qdbus $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.openDocumentSimple $1)
+    RESULT=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.openDocumentSimple $1)
     if ! [ "$RESULT" == "true" ]; then
         echo "Failed to open $1"
     fi
@@ -264,7 +264,7 @@
 
 # Opens a document in internally in the application
 function openDocuments {
-    RESULT=$(qdbus $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.openDocumentsSimple "(" $1 ")")
+    RESULT=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.openDocumentsSimple "(" $1 ")")
     if ! [ "$RESULT" == "true" ]; then
         echo "Failed to open $1"
     fi
@@ -278,7 +278,7 @@
     if ! [ "$WD" ]; then
         WD=$(pwd)
     fi
-    RESULT=$(qdbus $KDEV_DBUS_ID /org/kdevelop/ExternalScriptPlugin org.kdevelop.ExternalScriptPlugin.executeCommand "$CMD" "$WD")
+    RESULT=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/ExternalScriptPlugin org.kdevelop.ExternalScriptPlugin.executeCommand "$CMD" "$WD")
     if ! [ "$RESULT" == "true" ]; then
         echo "Execution failed"
     fi
@@ -292,26 +292,26 @@
     if ! [ "$WD" ]; then
         WD=$(pwd)
     fi
-    RESULT=$(qdbus $KDEV_DBUS_ID /org/kdevelop/ExternalScriptPlugin org.kdevelop.ExternalScriptPlugin.executeCommandSync "$CMD" "$WD")
+    RESULT=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/ExternalScriptPlugin org.kdevelop.ExternalScriptPlugin.executeCommandSync "$CMD" "$WD")
     echo "$RESULT"
 }
 
 # Getter functions:
 
 function getActiveDocument {
-    qdbus $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.activeDocumentPath $@
+    qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.activeDocumentPath $@
 }
 
 function getOpenDocuments {
-    qdbus $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.activeDocumentPaths
+    qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/DocumentController org.kdevelop.DocumentController.activeDocumentPaths
 }
 
 function raise! {
-    qdbus $KDEV_DBUS_ID /kdevelop/MainWindow org.kdevelop.MainWindow.ensureVisible
+    qdbus-qt5 $KDEV_DBUS_ID /kdevelop/MainWindow org.kdevelop.MainWindow.ensureVisible
 }
 
 function bdir! {
-    TARG=$(qdbus $KDEV_DBUS_ID /org/kdevelop/ProjectController org.kdevelop.ProjectController.mapSourceBuild "$(pwd)" false)
+    TARG=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/ProjectController org.kdevelop.ProjectController.mapSourceBuild "$(pwd)" false)
     if [ "$TARG" ]; then
         cd $TARG
     else
@@ -320,7 +320,7 @@
 }
 
 function project! {
-    TARG=$(qdbus $KDEV_DBUS_ID /org/kdevelop/ProjectController org.kdevelop.ProjectController.mapSourceBuild "$(pwd)" true)
+    TARG=$(qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/ProjectController org.kdevelop.ProjectController.mapSourceBuild "$(pwd)" true)
     if [ "$TARG" ]; then
         cd $TARG
     else
@@ -332,7 +332,7 @@
 # Main functions:
 
 function raise! {
-    qdbus $KDEV_DBUS_ID /kdevelop/MainWindow org.kdevelop.MainWindow.ensureVisible
+    qdbus-qt5 $KDEV_DBUS_ID /kdevelop/MainWindow org.kdevelop.MainWindow.ensureVisible
 }
 
 function sync! {
@@ -552,7 +552,7 @@
         LOCATION="$LOCATION;$(mapFileToClient $LOC)"
     done
 
-    qdbus $KDEV_DBUS_ID /org/kdevelop/GrepViewPlugin org.kdevelop.kdevelop.GrepViewPlugin.startSearch "$PATTERN" "$LOCATION" true
+    qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/GrepViewPlugin org.kdevelop.kdevelop.GrepViewPlugin.startSearch "$PATTERN" "$LOCATION" true
 }
 
 function dsearch! {
@@ -581,7 +581,7 @@
         LOCATION="$LOCATION;$(mapFileToClient $LOC)"
     done
 
-    qdbus $KDEV_DBUS_ID /org/kdevelop/GrepViewPlugin org.kdevelop.kdevelop.GrepViewPlugin.startSearch "$PATTERN" "$LOCATION" false
+    qdbus-qt5 $KDEV_DBUS_ID /org/kdevelop/GrepViewPlugin org.kdevelop.kdevelop.GrepViewPlugin.startSearch "$PATTERN" "$LOCATION" false
 }
 
 ##### SSH DBUS FORWARDING --------------------------------------------------------------------------------------------------------------------
