From 54a30606abc65311e54bce8a649fecb69baccf77 Mon Sep 17 00:00:00 2001
From: Atsushi Togo <atz.togo@gmail.com>
Date: Tue, 7 Nov 2017 20:18:09 +0900
Subject: [PATCH] Update src/Makefile.am to add determination.{c,h}

---
 src/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index ef1656e..4db45da 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,10 +1,11 @@
-lib_LTLIBRARIES = libsymspg.la 
+lib_LTLIBRARIES = libsymspg.la
 
 spglib_c = \
 arithmetic.c \
 cell.c \
 debug.c \
 delaunay.c \
+determination.c \
 hall_symbol.c \
 kgrid.c \
 kpoint.c \
@@ -26,6 +27,7 @@ arithmetic.h \
 cell.h \
 debug.h \
 delaunay.h \
+determination.h \
 hall_symbol.h \
 kgrid.h \
 kpoint.h \
