--- src/lib/OpenEXRCore/internal_cpuid.h.orig	2023-08-13 20:39:41.314691465 +0200
+++ src/lib/OpenEXRCore/internal_cpuid.h	2023-08-13 20:40:26.012689404 +0200
@@ -39,14 +39,13 @@
     *f16c = 0;
 #    endif
 
-#elif OPENEXR_ENABLE_X86_SIMD_CHECK
-
+#elif defined(__AVX__) && defined(__F16C__)
     // shortcut if everything is turned on / compiled in
-#    if defined(__AVX__) && defined(__F16C__)
     *f16c = 1;
     *avx = 1;
     *sse2 = 1;
-#    elif defined(_MSC_VER) && defined(_WIN32)
+#elif OPENEXR_ENABLE_X86_SIMD_CHECK
+#   if defined(_MSC_VER) && defined(_WIN32)
     int regs[4], osxsave;
 
     __cpuid (regs, 0);
