--- plasma-workspace-5.8.3/applets/digital-clock/package/contents/ui/DigitalClock.qml.orig	2016-11-10 20:47:59.739816619 +0100
+++ plasma-workspace-5.8.3/applets/digital-clock/package/contents/ui/DigitalClock.qml	2016-11-10 20:49:17.219813847 +0100
@@ -408,6 +408,7 @@
                 family: plasmoid.configuration.fontFamily || theme.defaultFont.family
                 weight: plasmoid.configuration.boldText ? Font.Bold : theme.defaultFont.weight
                 italic: plasmoid.configuration.italicText
+                pixelSize: 1024
                 pointSize: 1024
             }
 
@@ -432,6 +433,7 @@
 
             font.weight: timeLabel.font.weight
             font.italic: timeLabel.font.italic
+            font.pixelSize: 1024
             font.pointSize: 1024
             minimumPixelSize: 0
 
@@ -449,6 +451,7 @@
         font.family: timeLabel.font.family
         font.weight: timeLabel.font.weight
         font.italic: timeLabel.font.italic
+        font.pixelSize: 1024
         font.pointSize: 1024
         minimumPixelSize: 0
 
@@ -464,6 +467,7 @@
         font.family: timeLabel.font.family
         font.weight: timeLabel.font.weight
         font.italic: timeLabel.font.italic
+        font.pixelSize: 1024
         font.pointSize: 1024
 
         verticalAlignment: Text.AlignVCenter
