--- calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp.orig	2018-12-07 21:15:08.609967924 +0100
+++ calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp	2018-12-07 21:15:53.407972204 +0100
@@ -402,7 +402,7 @@
 
     QString str;
 
-    const char * p = s->getCString();
+    const char * p = s->c_str();
     int len = s->getLength();
     CharCode code;
     Unicode *u = nullptr;
@@ -451,11 +451,11 @@
     *d->body << " y=\"" << y << "px\"";
 
     if (font && font->getFamily()) {
-        *d->body << " font-family=\"" << QString::fromLatin1(font->getFamily()->getCString()) << "\"";
-        //debugPdf << "font family:" << QString::fromLatin1( font->getFamily()->getCString() );
+        *d->body << " font-family=\"" << QString::fromLatin1(font->getFamily()->c_str()) << "\"";
+        //debugPdf << "font family:" << QString::fromLatin1( font->getFamily()->c_str() );
     } else if (font && font->getName()) {
-        *d->body << " font-family=\"" << QString::fromLatin1(font->getName()->getCString()) << "\"";
-        //debugPdf << "font name:" << QString::fromLatin1( font->getName()->getCString() );
+        *d->body << " font-family=\"" << QString::fromLatin1(font->getName()->c_str()) << "\"";
+        //debugPdf << "font name:" << QString::fromLatin1( font->getName()->c_str() );
     }
     *d->body << " font-size=\"" << qMax(state->getFontSize(), state->getTransformedFontSize()) << "px\"";
 
