--- calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp.orig	2019-01-09 22:59:54.575623548 +0100
+++ calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp	2019-01-09 23:00:08.586624886 +0100
@@ -488,7 +488,7 @@
     if (maskColors) {
         for (int y = 0; y < height; y++) {
             dest = (unsigned int *)(buffer + y * 4 * width);
-            Guchar * pix = imgStr->getLine();
+            uchar * pix = imgStr->getLine();
             colorMap->getRGBLine(pix, dest, width);
 
             for (int x = 0; x < width; x++) {
@@ -507,7 +507,7 @@
     } else {
         for (int y = 0; y < height; y++) {
             dest = (unsigned int *)(buffer + y * 4 * width);
-            Guchar * pix = imgStr->getLine();
+            uchar * pix = imgStr->getLine();
             colorMap->getRGBLine(pix, dest, width);
         }
 
