--- calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp.orig	2019-11-02 14:44:28.386760151 +0100
+++ calligra-3.1.0/filters/karbon/pdf/SvgOutputDev.cpp	2019-11-02 14:46:06.221487282 +0100
@@ -405,7 +405,7 @@
     char * p = s->getCString();
     int len = s->getLength();
     CharCode code;
-    Unicode *u = nullptr;
+    const Unicode *u = nullptr;
     int uLen;
     double dx, dy, originX, originY;
     while (len > 0) {
